docs/modules/ROOT/pages/maven-configuration.adoc

```diff
@@ -0,0 +1,134 @@
+(https://maven.apache.org/settings.html)[see for more information the settings page of the official Maven
+website,role=external,window=_blank].
+
+1.  Create a `settings.xml` file in the `.m2` directory within your user directory.
+* If the directory does not exist, create it first: `mkdir ~/.m2`
+* You may have to set your computer to display invisible files and folders to see this directory.
+2. Copy the following into your new `settings.xml` file:
+
+[source,xml]
+----
+<settings xmlns="http://maven.apache.org/SETTINGS/1.0.0"
+          xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+          xsi:schemaLocation="http://maven.apache.org/SETTINGS/1.0.0
+                          http://maven.apache.org/xsd/settings-1.0.0.xsd">
+  <profiles>
+    <profile>
+      <id>standard-extra-repos</id>
+      <activation>
+        <activeByDefault>true</activeByDefault>
+      </activation>
+      <repositories>
+        <repository>
+          <id>archetype</id>
+          <url>ENTER YOUR MAVEN REPOSITORY HERE</url>
+          <releases>
+            <enabled>true</enabled>
+            <checksumPolicy>fail</checksumPolicy>
+          </releases>
+          <snapshots>
+            <enabled>true</enabled>
+            <checksumPolicy>fail</checksumPolicy>
+          </snapshots>
+        </repository>
+        <repository>
+          <id>GIVE YOUR REPOSITORY AN ID</id>
+          <name>GIVE YOUR REPOSITORY A NAME</name>
+          <releases>
+            <enabled>false</enabled>
+          </releases>
+          <snapshots>
+            <enabled>true</enabled>
+            <checksumPolicy>fail</checksumPolicy>
+          </snapshots>
+          <url>ENTER YOUR MAVEN REPOSITORY HERE</url>
+          <layout>default</layout>
+        </repository>
+      </repositories>
+    </profile>
+  </profiles>
+  <activeProfiles>
+    <activeProfile>standard-extra-repos</activeProfile>
+  </activeProfiles>
+</settings>
+----
+
+=== Maven Security Settings (OPTIONAL)
+
+NOTE: The below Maven security settings are recommended if you are working on a project with its own private
+repository.
+
+1. Open a terminal or command line and create a master password to seed password encryption capabilities.
+2. Type: `mvn -emp`
+* You should see a message similar to the following: `You will be prompted for the password.`
+* Enter your chosen password
+* You will receive an encrypted master password. Copy the encrypted master password.
+3. Create a `settings-security.xml` file within your `.m2` directory and substitute your encrypted master password in
+this file. **You must include the curly braces.** An example of the content is shown below.
+
+[source,xml]
+----
+  <settingsSecurity>
+    <master>INSERT YOUR ***ENCRYPTED*** MASTER PASSWORD HERE</master>
+  </settingsSecurity>
+----
+
+4. Encrypt the password that you will use to authenticate against the aiSSEMBLE Nexus repository:
```

Comment:
A: This number is off in the published page. It starts over and instead of saying 4 and 5, it says 1 and 2.